tests/test_create.py

```diff
@@ -491,29 +491,32 @@ def test_list_with_pip_no_binary(self):
     def test_list_with_pip_wheel(self):
         from conda.exports import rm_rf as _rm_rf
 
-        py_ver = "3.7"
+        py_ver = "3.10"
```

Comment:
I've also spent a lot of time trying to update these versions. It is valid to test that conda can install ancient Python, and that isn't a real problem until "python < old" has never been built for e.g. your ARM laptop.

Reply:
It might be useful to look at https://github.com/conda/conda/pull/12079 to enable auto-testing on M1